install.sh

```diff
@@ -1,45 +1,135 @@
 #!/bin/bash
 
-NET_SUBNET="192.168.231"
+
+# Copy the configuration file
+COPY='rsync -uv ' # Just because some platforms don't know
+				  # `cp -u`!
+S_PLATFORM=`uname`
+IS_LINUX=
+
+if [ "${S_PLATFORM}" == 'Linux' ]; then
+	COPY='cp -uv '
+	IS_LINUX=1
+fi
+
+$COPY -b ./.env.dist ./.env
+if [ ! -z "${IS_LINUX}" ]; then
+	sed -i -E "s|USER_ID=(.*)|USER_ID=$(id -u)\:$(id -g)|g" ./.env
```

Comment:
would be good to know what this is doing with some meaningful variables or comments